e2e-tests/tfm/cypress/e2e/journeys/case/deal-cancellation-enabled/deal-MIA.spec.js

```diff
@@ -0,0 +1,114 @@
+import format from 'date-fns/format';
+import relative from '../../../relativeURL';
+import pages from '../../../pages';
+import { caseSummary, caseSubNavigation } from '../../../partials';
+import MOCK_DEAL_MIA from '../../../../fixtures/deal-MIA';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../../e2e-fixtures';
+import { DATE_FORMATS } from '../../../../fixtures/constants';
+
+context('User can view a case deal', () => {
+  let dealId;
+  let dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_MIA, BANK1_MAKER1).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_MIA;
+
+      cy.createFacilities(dealId, mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities = createdFacilities;
+      });
+
+      cy.submitDeal(dealId, dealType, T1_USER_1);
+    });
+  });
+
+  beforeEach(() => {
+    cy.login(T1_USER_1);
+    cy.visit(relative(`/case/${dealId}/deal`));
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN);
+    dealFacilities.forEach(({ _id }) => {
+      cy.deleteFacility(_id, BANK1_MAKER1);
+    });
+  });
+
+  it('should render case deal components', () => {
+    caseSummary.container().should('exist');
+    caseSubNavigation.container().should('exist');
+    pages.caseDealPage.dealBankDetails().should('exist');
+    pages.caseDealPage.dealFacilities().should('exist');
+  });
+
+  it('should not show the deal cancellation button for PIM user', () => {
+    cy.login(PIM_USER_1);
+    cy.visit(relative(`/case/${dealId}/deal`));
+
+    pages.caseDealPage.cancelButton().should('not.exist');
+  });
+
+  it('should render case summary fields', () => {
```

Comment:
In other words, for example in this test, I don't think we need to test that the deal submission type is rendered. Because that's something that doesn't change if the feature flag is on/off